app/gateway/2.7.x/configure/auth/kong-manager/oidc.md

```diff
@@ -17,25 +17,26 @@ manually enable the **Plugin**; the configuration alone will enable
 The following is an example using Google as the IdP and serving Kong Manager
```

Comment:
[Concern]: if I am a new customer, I think I will not able to be refer to the "oidc authenticated_group" page, so I might confuse on the role mapping.

Reply:
In `oidc-mapping.md` -- which is the file I was planning to keep, and get rid of this one -- we have:

"The `authenticated_groups_claim` value specifies which IdP role should be assigned to the
   specified user. This value must be specified in the format provided in the example."

Is this enough?

Reply:
yes, I think that's better.